Source/AwsCommonRuntimeKit/io/HostAddress.swift

```diff
@@ -4,31 +4,21 @@
 import AwsCIo
 
 public struct HostAddress {
-    let host: String?
-    let address: String?
-    let recordType: AddressRecordType
-    let expiry: UInt64
-    let useCount: Int
-    let connectionFailureCount: Int
-    let weight: UInt8
+    public var host: String
+    public var address: String
+    public var recordType: AddressRecordType
```

Comment:
Any particular reason to define these as `var`s? If not, can we make them `let`s?

Reply:
These are var For consistency. We decided to keep all struct members as var instead of forcing consumers to create new structs if they want to update a value.